3days/data/src/main/java/com/weave/data/datasource/AuthRemoteDataSourceImpl.kt

```
@@ -1,60 +1,82 @@
+        authCodeId: UUID,
+        verifyCodeRequest: VerifyCodeRequest
+    ): NetworkResult<ExistingUserVerifyCodeResponse> {
+        return try {
+            val response = service.existingUserVerifyCode(authCodeId, verifyCodeRequest)
+
+            if(response.isSuccessful){
+                NetworkResult.Success(response.body()!!)
+            } else {
+                NetworkResult.Error(NetworkError.getNetworkErrorByCode(response.code()))
+            }
+        } catch (e: Exception){
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)